entrypoint.sh

```diff
@@ -11,6 +11,10 @@ if [[ -z "$PAGES_BRANCH" ]]; then
     PAGES_BRANCH="gh-pages"
 fi
 
+if [[ -z "$SOURCE_BRANCH_NAME" ]]; then
+    SOURCE_BRANCH_NAME="main"
```

Comment:
Can you change the default branch name to `master` instead of `main` as it would otherwise break the action for existing users.

Reply:
Please update the docs here as well: https://github.com/shalzz/zola-deploy-action/pull/57/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R9-R99